cli/spec/support/output_helpers.rb

```diff
@@ -52,23 +54,25 @@ module OutputHelpers
         line = 0
         @real.zip(@expected) do |real, expected|
           line += 1
-          error = values_match?(real.split(/\S+/), expected)
-          if error
-            @error ||= ""
-            @error << "\n#{error}"
-            @error.strip!
+          fields = real.split(/\s{2,}/)
```

Comment:
Maybe there's some way to trick tty-table to use a formatter that adds some recognizable separators, such as `\0` or `\t`. 

Perhaps that should be an option anyway, something like `-0|--print0` to output a null separated table that would be a lot easier to `cut` too.